pkg/amqp-bunny/AmqpSubscriptionConsumer.php

```diff
@@ -47,7 +47,7 @@ public function consume(int $timeout = 0): void
         try {
             $this->context->getBunnyChannel()->getClient()->run(0 !== $timeout ? $timeout / 1000 : null);
         } catch (ClientException $e) {
-            if ('stream_select() failed.' == $e->getMessage() && $signalHandler->wasThereSignal()) {
+            if ('stream_select() failed' === substr($e->getMessage(), 0, 22) && $signalHandler->wasThereSignal()) {
```

Comment:
would it be better to do `0 === strpos` to make sure it is the beginning of the exception message.